packages/metascraper-pdf/README.md

```
@@ -0,0 +1,124 @@
+//   title: 'Attention Is All You Need',
+//   author: 'Ashish Vaswani',
+//   publisher: 'arXiv',
+//   date: '2017-06-01T00:00:00.000Z',
+//   description: 'The dominant sequence transduction models are based on…',
+//   lang: 'en',
+//   logo: 'https://www.google.com/s2/favicons?domain_url=…',
+//   image: null
+// }
+```
+
+The bundle is a no-op unless [`test`](#test) sees a PDF URL, so it is safe to mix with the HTML rules:
```

markdownlint-cli2 (0.23.2)

[warning] Link fragments should be valid

(MD051, link-fragments)